src/d3d11/d3d11-device.cpp

```
@@ -285,6 +295,7 @@ Result DeviceImpl::initialize(const DeviceDesc& desc, BackendImpl* backend)
     if (m_info.timestampFrequency > 0)
     {
         addFeature(Feature::TimestampQuery);
```

Cppcheck (2.20.0)

[style] The function 'attach' is never used.

(unusedFunction)